python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -396,7 +404,7 @@ def assemble(self, skyInfo, tempExpRefList, imageScalerList, weightList, bgInfoL
             bit = afwImage.MaskU.getMaskPlane(plane)
             statsCtrl.setMaskPropagationThreshold(bit, threshold)
 
-        if self.config.doSigmaClip:
+        if doClip:
```

Comment:
I see no other reference to the `doSigmaClip` configuration option -- it looks like this change hard-codes it to be ignored. I suspect that's a mistake; if so, please fix it. If not, please justify it and remove the config option.